assets/quick-order-form.css

```diff
@@ -6,6 +6,22 @@
   background-color: rgba(var(--color-foreground), 0.02);
```

Comment:
Is this hover behaviour supposed to be here? 🤔 I dont think so

https://screenshot.click/02-09-eju1a-xyvsh.mp4

Reply:
There is no hover effect in here, this is a background for thead.

Reply:
When I hover in the video, the background color changes

Reply:
Oh, but this is what we have on Figma. According to the design all variant rows should have hover effect.

Reply:
Given the rows aren't clickable, Im not sure if they should have a hover 🤔 cc @melissaperreault

Reply:
I think it's a common approach and in this case it's juts for visual convenience, that way it's easy to see when you for example want to walk through variants and see what have you added. 

Let's @melissaperreault decide but I personally like it 🙂

Reply:
Let's keep the highlight! It REALLy helps to identify which variant line yo are interacting with even though it's not clickable. It helps the way finding a lot.

We'll have to adjust the padding on both sides though, something I can take a look closely to provide better guidance.